### DISCUSSION TITLE
Mirth Administrator Launcher 1.2.0?

### DISCUSSION THREAD

**Discussion by johns-1966** [nextgenhealthcare/connect#4518] [Q&A] [answered]
Is there an Administrator Launcher 1.2.0?  I have seen a prompt when starting the launcher to upgrade to 1.2.0.  Looking at the Nextgen site though I do not see a 1.2.0 on the download pages (still 1.1.0).

**Comment by narupley** [comment, marked as answer]
Yep there is, mainly to address #4379.

Our website team is still in the process of updating that page...

**Reply by johns-1966** [reply]
Thank you for the info.  It has gotten to where I don't trust anything, and then the download page only showed the old version.
